crates/iota-genesis-builder/src/lib.rs

```diff
@@ -907,6 +960,8 @@ impl Builder {
                 .save(file)?;
         }
 
+        // todo: probably need to save the delegator and delegator_map?
```

Comment:
Yes! 

For the delegator it can be something like the `serde_json::to_string(&self.migration_sources)` above. 

For the map you can use the `to_csv`method.

Reply:
done